qiskit/qobj/common.py

```diff
@@ -0,0 +1,83 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+# pylint: disable=invalid-name
+
+"""Module providing definitions of common Qobj classes."""
+import json
+import os
+from types import SimpleNamespace
+
+import fastjsonschema
+
+
+path_part = 'schemas/qobj_schema.json'
+path = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+    path_part)
+with open(path) as fd:
+    json_schema = json.loads(fd.read())
+validator = fastjsonschema.compile(json_schema)
```

Comment:
Moving this here is potentially problematic. I know of downstream consumers that are using this in its previous location

Reply:
Nevermind this was my mistake, as you pointed out it's being reimported in `qasm_qobj` I completely missed that and just concentrated on the removed line.